From 50804eb801ed4a29c899d706cef05a72397e34e8 Mon Sep 17 00:00:00 2001
From: Gustavo Duarte <gduarte@redhat.com>
Date: Mon, 13 Oct 2014 22:30:16 -0400
Subject: [redhat] configs: Enable GenWQE config options

Message-id: <1413239416-7778-29-git-send-email-gduarte@redhat.com>
Patchwork-id: 97553
O-Subject: [RHEL7.1 PATCH BZ 1088491 28/28 v2] Enable GenWQE config options
Bugzilla: 1088491
RH-Acked-by: Don Zickus <dzickus@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1088491
Upstream Status: RHEL only
---
 create mode 100644 redhat/configs/generic/powerpc64/CONFIG_GENWQE
 create mode 100644 redhat/configs/generic/powerpc64/CONFIG_GENWQE_PLATFORM_ERROR_RECOVERY

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/configs/generic/powerpc64/CONFIG_GENWQE b/redhat/configs/generic/powerpc64/CONFIG_GENWQE
new file mode 100644
index 0000000..f1f996a
--- /dev/null
+++ b/redhat/configs/generic/powerpc64/CONFIG_GENWQE
@@ -0,0 +1 @@
+CONFIG_GENWQE=m
diff --git a/redhat/configs/generic/powerpc64/CONFIG_GENWQE_PLATFORM_ERROR_RECOVERY b/redhat/configs/generic/powerpc64/CONFIG_GENWQE_PLATFORM_ERROR_RECOVERY
new file mode 100644
index 0000000..b14ba84
--- /dev/null
+++ b/redhat/configs/generic/powerpc64/CONFIG_GENWQE_PLATFORM_ERROR_RECOVERY
@@ -0,0 +1 @@
+CONFIG_GENWQE_PLATFORM_ERROR_RECOVERY=1
-- 
1.7.1